xblockutils/studio_editable.py

```diff
@@ -0,0 +1,295 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2015 OpenCraft
```

Comment:
@sarina This comes from XBlocks where there is no requirement to assign the copyright, but it's true that since this is contributed to a repository on the edx organization, it probably should be assigned to edX and have (c) edX everywhere. 

Not sure why there isn't a copyright notice on the edx-platform though, as this is a standard good practice - maybe nobody got around to do it there when the code was released? @nedbat do you know if there is another reason?